griptape/drivers/prompt/anthropic_prompt_driver.py

```diff
@@ -21,9 +18,9 @@ class AnthropicPromptDriver(BasePromptDriver):
         tokenizer: Custom `AnthropicTokenizer`.
     """
 
-    api_key: str = field(kw_only=True, metadata={"serializable": True})
+    api_key: Optional[str] = field(kw_only=True, default=None, metadata={"serializable": True})
```

Comment:
for my own knowledge: why mark this as `Optional` AND set a default? just setting a default will functionally make it optional, wouldn't it?